### PR TITLE
29 blogpost benchmark of hashing algorithms tutorial

### PR DESCRIPTION
**Pull request by alv-around**
https://github.com/ZoKratesPlus/zokratesplus.github.io/issues/29

### CODE REVIEW

**Comment by alvaro-alonso** [comment]
@JonathanHeiss @nikiblume Here is the material related to the blogpost comparing different[ hashing algorithms](https://github.com/ZoKratesPlus/zokratesplus.github.io/issues/29). Can one of you check it out, and let me know if something is unclear ? :pray:

**Comment by alvaro-alonso** [comment]
> @JonathanHeiss @nikiblume Here is the material related to the blogpost comparing different[ hashing algorithms](https://github.com/ZoKratesPlus/zokratesplus.github.io/issues/29). Can one of you check it out, and let me know if something is unclear ? 🙏

@nikiblume can you please review the branch?

File: hashes/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      > 6 195 173 103 172 169 108 58 114 105 178 230 75 24 7 ...
+      ``` 
+      or 
+      ```sh 
+      $ npm install
+      $ node witness.js
+      ```
+   2. Add the output generated by `witness.py` as arguments in ZoKrates
+      ```sh
+      $ zokrates compute-witness -a 6 195 173 103 172 169 108 58 114 105 178 230 75 24 7 ...
```
Comment (nikiblume) [inline hashes/README.md:41]: I would either add to the README that for poseidon you need for witness generation as input: (hash, 1, 2), or be consistent and change `witness.js` to also give the full inputs for the `hash.zok` like done for the blake2s hash.
=> I think the current state could confuse newbies

File: hashes/poseidon/witness.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/node
+const circomlibjs = require("circomlibjs");
+
+(async function () {
+    const preimage = [1, 2]
+    const poseidon = await circomlibjs.buildPoseidon();
+    const hash = poseidon.F.toString(poseidon(preimage));
+    console.log(`poseidon hash of preimage(${preimage}) is: ${hash}`);
```
Comment (nikiblume) [inline hashes/poseidon/witness.js:8]: for consistency and more easy use, I would make the output of `witness.js` the same as in `witness.py` in the poseidon example. Meaning to have the hash and the preimage. This way the user can directly use the output as input for the witness generation, without having to add the preimage manually

File: hashes/sha256/hash.zok [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import "hashes/sha256/sha256" as sha256;
```
Comment (nikiblume) [inline hashes/sha256/hash.zok:1]: compiler error: wrong sha256 function used => hashes/sha256/512bitPadded is correct for these inputs

**Review by nikiblume — COMMENTED** [review]
Tested the whole /hashes repo. One compiler issue and consistency improvement suggestion

**Pull request merged**